internal/cloud/realm/auth.go

```diff
@@ -97,6 +97,25 @@ func (c *client) getAuth(options api.RequestOptions) (string, error) {
 	return "", nil
 }
 
+//Refresh Auth uses the user's refresh token to attempt to create a new access token
+func (c *client) refreshAuth() (string, error) {
+	res, resErr := c.do(http.MethodPost, refreshAuthPath, api.RequestOptions{RefreshAuth: true})
+
+	if resErr != nil {
+		return "", resErr
+	}
+	if res.StatusCode != http.StatusCreated {
+		return "", ErrInvalidSession
+	}
+	defer res.Body.Close()
+
+	var session Session
+	if err := json.NewDecoder(res.Body).Decode(&session); err != nil {
+		return "", err
+	}
+	return session.AccessToken, nil
```

Comment:
I think we want this to return the session, ultimately once we're able to save the new details on the profile we'll probably want to save both access and refresh tokens

Reply:
The session endpoint appears to only return the access token (unless I completely whiffed on something). Although I can see the confusion since we reuse the session object here. If there is a way to get the refresh token, then I can make that fix

Reply:
ah gotcha, yeah that makes sense.  feel free to leave as is!